src/futil.js

```diff
@@ -3,4 +3,34 @@ import F from 'futil'
 
 export let flattenProp = _.curry((prop, target) =>
   _.flow(F.expandObject(_.get(prop)), _.unset(prop))(target)
-)
\ No newline at end of file
+)
+
+// See R.evolve, but supports lodash nested paths
+// NOTE: does not run on objects where path is missing. flow with defaults/ensureKeys if desired
+// CAVEAT: When using `_.curry`, fast-csv's transform doesn't work
+export let updateMany = rules => data => {
+  let clone = _.cloneDeep(data)
+  F.eachIndexed((display, field) => {
+    if (_.has(field, clone))
+      F.updateOn(field, value => display(value, clone), clone)
+  })(F.compactObject(rules))
+  return clone
+}
+
+// F.arrayToObject with keys as array values
+// (['a', 'b'], x => x + 'c') => { a: 'ac', b: 'bc' }
+export let keysToObject = _.curry(
+  (toValue, data) => F.arrayToObject(x => x, toValue, data)
```

Comment:
You can just use `_.identity` anywhere you would use `x => x` since it's already an established convention in our repos